neo/rawio/plexon2rawio/pypl2/pypl2lib.py

```diff
@@ -8,22 +8,26 @@
 # You are free to modify or share this file, provided that the above
 # copyright notice is kept intact.
 
-from sys import platform
-import os
+import platform
+import subprocess
 import pathlib
 import warnings
+import numpy as np
 
-if any(platform.startswith(name) for name in ("linux", "darwin", "freebsd")):
-    from zugbruecke import CtypesSession
+platform_is_windows = platform.system() == "Windows"
 
-    ctypes = CtypesSession(log_level=100)
-
-elif platform.startswith("win"):
+if platform_is_windows:
     import ctypes
 else:
-    raise SystemError("unsupported platform")
```

Comment:
Could this ever happen? Are there other non-posix systems that we should put as an else?

Reply:
I thought that it was unecessary but it does not hurt. Do you want me to add just in case?

Reply:
Might as well just in case. I'm sure someone is using something crazy and just itching to try and fail to read .pl2 files.

Reply:
Yeah, now that I think about it I don't think it fits the current branch structure. You want the second branch for something else. I rather have someone come with that problem and then we fix it.

Reply:
That's fair. I mean they will still get an error. They just won't get your nice error message.